lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvConnection.php

```diff
@@ -73,6 +75,7 @@ public function __construct($serverName, $connectionOptions)
             throw SQLSrvException::fromSqlSrvErrors();
         }
         $this->lastInsertId = new LastInsertId();
```

Comment:
Shouldn't this use Driver\LastInsertId too instead of Driver\SQLSrv\LastInsertId? For BC, Driver\SQLSrv\LastInsertId can exend Driver\LastInsertId and be deprecated.